.mergify.yml

```
@@ -1,7 +1,8 @@
 # Linear queue for the main branch.
 queue_rules:
   - name: main
-    conditions:
+    commit_message_template: "Merge (#{{ number }}): {{ title }}\n\n{{ body }}"
```

I'd prefer something like:

```suggestion
    commit_message_template: "{{ title }} (#{{ number }})\n\n{{ body | get_section("## Description") }}"
```

We can edit our issue template to include an HTML comment (which is automatically stripped) that any content of the `Description` section will be used as the commit body (along the the PR title)

---

I'd like to keep Merge as the first word to make clear in UIs that it's a merge commit. Why remove it?

I think it's important that the whole body be included so it had the Security and Scaling considerations included.

This seems like a topic for more than the two of us so I'll bring up after the release.

---

> I'd like to keep Merge as the first word to make clear in UIs that it's a merge commit. Why remove it?

Because it would apply to squashed PRs too and it feels weird to have `Merge` for those. Why do you want to `Merge` prefix? Doesn't the PR number suffix make it clear in the UI already ?

> This seems like a topic for more than the two of us so I'll bring up after the release.

Agreed!